pincer/utils/directory.py

```diff
@@ -6,7 +6,7 @@
 
 @contextmanager
 def chdir(path) -> Generator[str, Any, None]:
-    """Change the cwd to the given path as a context manager."""
+    """Change the cwd to the given path as a context manager"""
```

Comment:
Can you add back the dot please? Also put it on the other ones ^^

> Quoted from [**PEP 257**](https://www.python.org/dev/peps/pep-0257/)

- *The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".*

Reply:
Yes, sure. TIL end docstrings with a dot.